src/main/java/com/hp/octane/plugins/bamboo/octane/BambooPluginServices.java

```diff
@@ -110,6 +117,34 @@ public BambooPluginServices() {
         this.accessor = ComponentLocator.getComponent(ResultsSummaryVariableAccessor.class);
         this.resultsSummaryManager = ComponentLocator.getComponent(BuildResultsSummaryManager.class);
         this.chainBranchManager = ComponentLocator.getComponent(ChainBranchManager.class);
+        this.buildLogFileAccessorFactory = ComponentLocator.getComponent(BuildLogFileAccessorFactory.class);
+    }
+
+    @Override
+    public InputStream getBuildLog(String jobId, String buildId) {
+        InputStream result = null;
+        BuildLogFileAccessor fileAccessor = null;
+        try {
+            PlanKey planKey = PlanKeys.getPlanKey(jobId);
+            int buildNumber = PlanKeys.getPlanResultKey(buildId).getBuildNumber();
```

Comment:
I think it need to add null check here
PlanKeys.getPlanResultKey(buildId) != null then continue with logic else print logs